docs/topics/time.md

```diff
@@ -0,0 +1,416 @@
+enum class:
+  * `NANOSECONDS`
+  * `MICROSECONDS`
+  * `MILLISECONDS`
+  * `SECONDS`
+  * `MINUTES`
+  * `HOURS`
+  * `DAYS`
+
+`Durations` can be positive, negative, zero, positive infinity, or negative infinity.
+
+### Create duration
+
+To create a `Duration`, use the [extension properties](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-duration/#companion-object-properties) 
+available for `Int`, `Long`, and `Double` types: `nanoseconds`, `microseconds`, `milliseconds`, `seconds`, `minutes`, 
+`hours`, and `days`.
+
+> Days refer to periods of 24 hours. They are not calendar days.
+> 
+{type = "tip"}
+
+For example:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.Duration.Companion.days
+
+fun main() {
+    //sampleStart
+    val fiveHundredMilliseconds: Duration = 500.milliseconds
+    val zeroSeconds: Duration = 0.seconds
+    val tenMinutes: Duration = 10.minutes
+    val negativeNanosecond: Duration = (-1).nanoseconds
+    val infiniteDays: Duration = Double.POSITIVE_INFINITY.days
+    val negativeInfiniteDays: Duration = Double.NEGATIVE_INFINITY.days
+
+    println(fiveHundredMilliseconds) // 500ms
+    println(zeroSeconds)             // 0s
+    println(tenMinutes)              // 10m
+    println(negativeNanosecond)      // -1ns
+    println(infiniteDays)            // Infinity
+    println(negativeInfiniteDays)    // -Infinity
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-create-duration"}
+
+You can also perform basic arithmetic with `Durations`:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+    //sampleStart
+    val fiveSeconds: Duration = 5.seconds
+    val thirtySeconds: Duration = 30.seconds
+
+    println(fiveSeconds + thirtySeconds)
+    // 35s
+    println(thirtySeconds - fiveSeconds)
+    // 25s
+    println(fiveSeconds * 2)
+    // 10s
+    println(thirtySeconds / 2)
+    // 15s
+    println(thirtySeconds / fiveSeconds)
+    // 6.0
+    println(-thirtySeconds)
+    // -30s
+    println((-thirtySeconds).absoluteValue)
+    // 30s
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-create-duration-arithmetic"}
+
+### Get string representation
+
+It can be useful to have a string representation of a `Duration` so that you can print, serialize, transfer, or store it.
+
+To get a string representation, use the `.toString()` function. By default, the time is reported using each unit that is
+present. For example: `1h 0m 45.677s` or `-(6d 5h 5m 28.284s)`
+
+To configure the output, use the `.toString()` function with your desired `DurationUnit` and number of decimal places
```

Comment:
```suggestion
To configure the output, use the `.toString()` function with your desired `DurationUnit` and a number of decimal places
```

Reply:
In this case, we don't need an article here. It's more natural without it.